src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
> I have done more for this community than most of the people with who I am interacting here will do in their lifetime. You may want to reflect on that too.

Reflecting on this I don't see how this gives you more authority to discuss technical issues about opam.

> Without Janestreet and OCamlPro, there would be no ocaml.org and no OPAM. Where do you see a sustainable environment without them ?

This is precisely the problem at the moment. We should strive for an environment that is able to survive without them and bring more companies in to invest in a company neutral ecosystem. OCamlPro would be the first to benefit from it. 

> By the way, OCamlPro was created to help the OCaml community, not the other way around.

I don't care about intentions, I care about actions. 

> So, OCamlPro should keep on creating new software and services, then move them to ocaml.org, so that people like you can believe that OCamlPro did not create them, and that they can disrespect us. Nice.

If you believe OCamlPro is the only entity creating value in this system and take objecting to hardcode OCamlPro URIs in opam as disrespect I'm afraid there's not much left to be discussed at that point.